### DISCUSSION TITLE
Wake/Shutdown device if specific ip responding to ping

### DISCUSSION THREAD

**Discussion by Nonobis** [seriousm4x/UpSnap#704] [Feature requests]
Hello,

Do you think it's possible to add an option to auto-wake a device if an ip is activated ?
For sample, wake up my emby server if 1 or more tv is pingable ?
and send sleep command if device is not responding ?

a system like the cron but with a list of ip to check ?

Thanks

**Comment by seriousm4x** [comment]
I'd prefer to not include this. It's a quite unique request. If it gains a lot of interest from others as well, I will reconsider.

For now, you can create your own script with the [api](https://github.com/seriousm4x/UpSnap/wiki/Rest-API).
